README.md

```diff
@@ -69,22 +73,22 @@ If the project is now up and running for you locally, you might want to make a s
 
 The system:
 
-    You should fork the project from the original repo.
-    Get it running locally.
-    For new work create a new branch with an appropriate name.
-    Make changes, and commit.
-    Push the changes to your repo.
-    Make a PR from your repo to the original.
-    Respond to correspondence about the PR (if needed).
-    Make changes (if needed).
-    And on you go.. 
+    1. You should fork the project from the original repo.
+    2. Get it running locally.
+    3. For new work create a new branch with an appropriate name.
+    4. Make changes, and commit.
+    5. Push the changes to your repo.
+    6. Make a PR from your repo to the original.
+    7. Respond to correspondence about the PR (if needed).
+    8. Make changes (if needed).
+    9. And on you go.. 
 
 A few other pointers:
 
-    Meaningful PR messages. These should be roughly aligned with the significance of the PR. For anything complex, be prepared to thoroughly explain in the PR what the aim of the PR is, and how you achieve that aim. Again, this is within reason, but for particularly thorny parts of the code, please explain anything that might need it. 
-    Add any tags to the PR that are needed (WIP, Spike, etc)
-    If the PR is from a Trello task, please add a link to that task.
-    If you make some git errors, don’t hesitate to have a chat with someone from the team.
+    - Meaningful PR messages. These should be roughly aligned with the significance of the PR. For anything complex, be prepared to thoroughly explain in the PR what the aim of the PR is, and how you achieve that aim. Again, this is within reason, but for particularly thorny parts of the code, please explain anything that might need it. 
+    - Add any tags to the PR that are needed (WIP, Spike, etc)
+    - If the PR is from a Trello task, please add a link to that task.
+    - If you make some git errors, don’t hesitate to have a chat with someone from the team.
```

Comment:
Please remove the indentation in each bullet point.

```suggestion
- Meaningful PR messages. These should be roughly aligned with the significance of the PR. For anything complex, be prepared to thoroughly explain in the PR what the aim of the PR is, and how you achieve that aim. Again, this is within reason, but for particularly thorny parts of the code, please explain anything that might need it. 
- Add any tags to the PR that are needed (WIP, Spike, etc)
- If the PR is from a Trello task, please add a link to that task.
- If you make some git errors, don’t hesitate to have a chat with someone from the team.
```

Reply:
Done.